inspect_gui.py

```
@@ -0,0 +1,232 @@
+            return
+        self._set_busy(True, "Rendering inspected artifact…")
+
+        def worker() -> None:
+            try:
+                chunks = render_schedule(
+                    self.source_var.get(),
+                    duration,
+                    allow_disclosed_changes=self.ack_var.get(),
+                )
+                result = write_audio(chunks, destination)
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)